docs/source/llm/getting-started.md

```diff
@@ -14,221 +14,308 @@
 
 ## Prerequisites
 
-Let’s start by getting an ExecuTorch environment:
+To follow this guide, you'll need to clone the ExecuTorch repository and install dependencies. 
+ExecuTorch requires Python 3.10, as well as  This example uses
+conda to manage the Python environment. 
 
-1.  Create a third-party folder (Keeps the file paths organized)
-```
-mkdir  third-party
-cd  third-party
-```
-2. If you’re new to ExecuTorch follow [these steps](https://pytorch.org/executorch/main/getting-started-setup.html#set-up-your-environment) to set up your environment.
+::::{tab-set}
+:::{tab-item} conda
+Instructions on installing miniconda can be [found here](https://docs.anaconda.com/free/miniconda).
```

Comment:
can we eliminate the need for Conda. 
Can you re-write this so that it uses env?

Reply:
O looks like you have both options here. Perfect. I'd change the instructions just a tad so it calls out two different options up front before giving the conda instructions